.env

```diff
@@ -0,0 +1,3 @@
+API_URL = "https://random-words-api-umber.vercel.app/word"
```

Comment:
Never ever make this mistake again.

DO NOT COMMIT OR PUSH `.env` TO THE CODE. Add it into .gitignore

Reply:
ok